tests/dragonfly/cluster_test.py

```diff
@@ -1858,10 +1905,26 @@ async def test_cluster_migration_huge_container(df_factory: DflyInstanceFactory)
     await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
 
     logging.debug("Waiting for migration to finish")
-    await wait_for_status(nodes[0].admin_client, nodes[1].id, "FINISHED")
+    await wait_for_status(nodes[0].admin_client, nodes[1].id, "FINISHED", timeout=30)
 
-    target_data = await StaticSeeder.capture(nodes[1].client)
-    assert source_data == target_data
+    if seed_during_migration:
+        await stop_seed()
+    else:
+        # Only verify memory growth if we haven't pushed new data during migration
```

Comment:
I am not sure we need this option of not running seeder during migration 
you added this option inorder to check the rss compared to rss before migration right? but we can compare peak rss to peak used memory

Reply:
more importantly is the comparison - we compare before and after if we don't seed during the migration

Before migrating to Seeder and comparison, I used custom logic for checking the data that was inserted. With the seeder I'm afraid it is no longer possible to check, unless you have an idea on how to do that.

I personally vote for keeping the previous logic, let me know if you want that as well..